api/v1alpha1/kafkatopic_types.go

```diff
@@ -40,7 +40,8 @@ type KafkaTopicSpec struct {
 // KafkaTopicStatus defines the observed state of KafkaTopic
 // +k8s:openapi-gen=true
 type KafkaTopicStatus struct {
-	State TopicState `json:"state"`
+	State     TopicState `json:"state"`
+	ManagedBy string     `json:"managedBy"`
```

Comment:
**Question**: What is the reason we are making the `managedBy` part of the status in addition to using it as an annotation? (Looks like double booking to me, but I'm mostly just curious, not debating it.)

Reply:
When there is no annotation the referenced kafka topic is managed by Koperator.
I think it is good to show who is the manager also in the topic status.
This can be a confirmation for the user when he checks the topic status that is managed by who.

Reply:
Since we're handling this in annotations everywhere else wouldn't it make more sense to add an annotation instead?